src/libstd/macros.rs

```diff
@@ -33,6 +33,8 @@
 /// panic!(4); // panic with the value of 4 to be collected elsewhere
 /// panic!("this is a {} {message}", "fancy", message = "message");
 /// ```
+///
+/// [`Box<Any>`]: ../../collections/boxed/struct.Box.html
```

Comment:
There is one too many `../`s for the links you've added in this file. The macros are documented at the crate root so you only need to go up once.

Also this one should really be linking to `std::boxed::Box` and not `collections::boxed::Box`.